static/js/webrtc.js

```diff
@@ -516,8 +553,68 @@ var rtc = (function() {
         return false;
       }
     },
+
+    // Connect a setting to a checkbox. To be called on initialization.
+    //
+    // It will check for the value in urlVar, cookie, and clientVar, in that order
+    // If urlVar is found, it will also set the cookie
+    // Finally, it sets up to set cookie if the user changes the setting in the gearbox
+    settingToCheckbox: function({urlVar, cookie, clientVar, checkboxId}) {
```

Comment:
This should be usable for any new checkbox that's tied to a clientVar, a urlVar, and a cookie the way these are. I plan to port the existing "A/V enabled" checkbox here as a followup PR. Should be cleaner.

Reply:
I will likely need to change this function definition, since destructuring like this may be somewhat modern Javascript.